samples/apps/copilot-chat-app/scripts/Configure.ps1

```diff
@@ -0,0 +1,133 @@
+<#
+.SYNOPSIS
+Configure user secrets, appsettings.Development.json, and .env for Copilot Chat.
+
+.PARAMETER OpenAI
+Switch to configure for OpenAI.
+
+.PARAMETER AzureOpenAI
+Switch to configure for Azure OpenAI.
+
+.PARAMETER Endpoint
+Set when using Azure OpenAI.
+
+.PARAMETER ApiKey
+The API key for the AI service.
+
+.PARAMETER CompletionModel
+The chat completion model to use (e.g., gpt-3.5-turbo or gpt-4).
+
+.PARAMETER EmbeddingModel
+The embedding model to use (e.g., text-embedding-ada-002).
+
+.PARAMETER PlannerModel
+The chat completion model to use for planning (e.g., gpt-3.5-turbo or gpt-4).
+
+.PARAMETER ClientID
+The client (application) ID associated with your AAD app registration.
+
+.PARAMETER Tenant
+The tenant (directory) associated with your AAD app registration.
+Defaults to 'common'. 
+See https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration#authority.
+#>
+
```

Comment:
#Requires -Version 6

Reply:
I think you need it for IsWindows/Mac/Linux

Reply:
oh yes - I see now.